ouroboros-consensus/test-consensus/Test/ThreadNet/Ref/PBFT.hs

```diff
@@ -0,0 +1,620 @@
+{-# LANGUAGE LambdaCase          #-}
```

Comment:
See the commits for a delta without the renaming (can't pass the `git diff -M` option to GitHub's PR renderer?).